File: Source/ntss-src/ntss-core/src/main/java/jp/co/nikkiso/ntss/core/entity/MstDeviceSetInfoDefault.java
package jp.co.nikkiso.ntss.core.entity;

import jp.co.nikkiso.ntss.core.entity.entityListener.CommonEntityListener;
import org.seasar.doma.Entity;
import org.seasar.doma.Id;
import org.seasar.doma.Table;
import org.seasar.doma.jdbc.entity.NamingType;

import lombok.Getter;
import lombok.Setter;


@Entity(listener = CommonEntityListener.class, naming = NamingType.NONE)
@Table(name = "mst_device_set_info_default")
@Getter
@Setter
public class MstDeviceSetInfoDefault extends BaseBlankEntity {
  @Id
  private String facility_cd;
  private String device_set_info;
  private String reg_date;
  private String up_date;
  /** 風袋補正情報 */
  private String tare_info;
  /** 除水補正情報 */
  private String off_water_info;
  /** ホスト報知情報 */
  private String host_notification_info;

}

File: Source/ntss-win/NKKWeight/Source/NKKWeightScaleDB/Models/Mst_device_set_info_default.cs
namespace NKKWeightScaleDB.Models
{
    using Newtonsoft.Json;

    public class Mst_device_set_info_default : BaseEntity
    {
        [JsonProperty(PropertyName = "facilityCd")]
        public string facility_cd { get; set; }

        [JsonProperty(PropertyName = "deviceInfo")]
        public string device_set_info { get; set; }

        [JsonProperty(PropertyName = "tare_info")]
        public string tare_info { get; set; }

        [JsonProperty(PropertyName = "off_water_info")]
        public string off_water_info { get; set; }

        [JsonProperty(PropertyName = "regDate")]
        public string reg_date { get; set; }

        [JsonProperty(PropertyName = "upDate")]
        public string up_date { get; set; }
    }
}